server/package.json

```diff
@@ -20,7 +20,7 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.2",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v10.2.0",
+    "brave-alert-lib": "github:bravetechnologycoop/brave-alert-lib#mdb-google-helpers",
```

Comment:
Just leaving a comment here as a reminder to go back and update this to the version number once the brave-alert-lib part is "deployed".